sky/clouds/ibm.py

```diff
@@ -415,8 +418,12 @@ def check_credentials(cls) -> Tuple[bool, Optional[str]]:
 
     @classmethod
     def check_disk_tier_enabled(cls, instance_type: str,
-                                disk_tier: str) -> None:
-        del instance_type, disk_tier  # unused
+                                disk_tier: resources_utils.DiskTier) -> None:
+        del instance_type  # unused
+        if disk_tier is None or disk_tier == resources_utils.DiskTier.BEST:
+            return
+        raise exceptions.NotSupportedError(
+            f'IBM does not support disk tier {disk_tier.value}.')
```

Comment:
Instead of creating a similar function for all the clouds, how about we add a variable in `Cloud._SUPPORTED_DISK_TIER: set[DiskTier]` and only have this function in `Cloud` by checking the `_SUPPORTED_DISK_TIER`?

Reply:
It would be good to have a default value for `_SUPPORTED_DISK_TIER = {DiskTier.BEST}`

Reply:
Good point! Done. Thanks!